src/App.js

```diff
@@ -1,127 +1,15 @@
-
-  // get access from Airtable
-  const fetchData = async () => {
-    const options = {
-      method: 'GET',
-      headers: {
-        Authorization: `Bearer ${process.env.REACT_APP_AIRTABLE_API_TOKEN}`,
-      },
-    };
-
-    try {
-      const response = await fetch(url, options);
-
-      if (!response.ok) {
-        const message = `Error: ${response.status}`;
-        throw new Error(message);
-      }
-
-      const data = await response.json();
-
-      const todos = data.records.map((todo) => {
-        const newTodo = {
-          id: todo.id,
-          title: todo.fields.title,
-        };
-        return newTodo;
-      });
-
-      setTodoList(todos);
-      setIsLoading(false);
-    } catch (error) {
-      console.log(error.message);
-    }
-  };
-
-  React.useEffect(() => {
-    fetchData();
-  }, []);
-
-  React.useEffect(() => {
-    if (!isLoading) {
-      localStorage.setItem('savedTodoList', JSON.stringify(todoList));
-    }
-  }, [todoList, isLoading]);
-
-  // to add an item to Airtable
-  const addTodo = async (todo) => {
-    const airtableData = {
-      fields: {
-        title: todo,
-      },
-    };
-
-    const options = {
-      method: 'POST',
-      headers: {
-        'Content-Type': 'application/json',
-        Authorization: `Bearer ${process.env.REACT_APP_AIRTABLE_API_TOKEN}`,
-      },
-      body: JSON.stringify(airtableData),
-    };
-
-    try {
-      const response = await fetch(url, options);
-
-      if (!response.ok) {
-        const message = `Error has ocurred: ${response.status}`;
-        throw new Error(message);
-      }
-
-      const resp = await response.json();
-      const newTodo = { id: resp.id, title: resp.fields.title };
-      setTodoList([...todoList, newTodo]);
-    } catch (error) {
-      console.log(error.message);
-      return null;
-    }
-  };
-
-  // remove an item from Airtable
-  const removeTodo = async (id) => {
-    const options = {
-      method: 'DELETE',
-      headers: {
-        'Content-Type': 'application/json',
-        Authorization: `Bearer ${process.env.REACT_APP_AIRTABLE_API_TOKEN}`,
-      },
-    };
-
-    try {
-      const response = await fetch(`${url}/${id}`, options);
-
-      if (!response.ok) {
-        const message = `Error has ocurred: ${response.status}`;
-        throw new Error(message);
-      }
-
-      const newList = todoList.filter((item) => item.id !== id);
-      setTodoList(newList);
-    } catch (error) {
-      console.log(error.message);
-      return null;
-    }
-  };
-
   return (
-    <div>
-      <h1>Todo List</h1>
-      <AddTodoForm onAddTodo={addTodo} />
-      {isLoading ? (
-        <p>Loading...</p>
-      ) : (
-        <TodoList todoList={todoList} onRemoveTodo={removeTodo} />
-      )}
-    </div>
+    <BrowserRouter>
+      <Routes>
+        <Route path="/" element={<ReactTodo />}></Route>
+        <Route path="/new" element={<h1>New Todo List</h1>}></Route>
```

Comment:
You could use react-router-dom Route component with an opening-closing tag when has some other nested routes, example:

```
<Route path="/dashboard" element={<Dashboard />}>
  <Route index element={<DashboardHome />} />
  <Route path="settings" element={<Settings />} />
</Route>
```

Otherwise (like in this case) you can use Route with a self-closing tag.